lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -6901,5 +6900,15 @@ is the only supported options.</xs:documentation>
       <xs:enumeration value="aggressive"/>
     </xs:restriction>
   </xs:simpleType>
-
+  <xs:simpleType name="CollectionType">
+    <xs:annotation>
+      <xs:documentation xml:lang="en">Allowed collection types</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
```

Comment:
I'm not sure how this needs to be specified, but this caused https://app.circleci.com/pipelines/github/galaxyproject/galaxy/20649/workflows/f38e227c-a4e1-4594-b5af-8cab9f4815fb/jobs/162241?invite=true#step-105-870 to fail.

Reply:
I see. It can be a comma separated list, at least in the test. Will go for a regular expression.